pkg/proc/gdbserial/gdbserver.go

```diff
@@ -423,9 +434,12 @@ func LLDBLaunch(cmd []string, wd string) (*Process, error) {
 		proc = exec.Command("lldb-server", args...)
 	}
 
-	if logflags.LLDBServerOutput() || logflags.GdbWire() {
+	if logflags.LLDBServerOutput() || logflags.GdbWire() || foreground {
 		proc.Stdout = os.Stdout
 		proc.Stderr = os.Stderr
+		if foreground {
```

Comment:
Let's just move this out of the enclosing branch and have it stand on it's own so we're not checking the boolean twice, the short circuit logic doesn't buy us anything and makes the code more verbose.